app/src/main/java/ch/epfl/sweng/swenggolf/tools/FragmentConverter.java

```diff
@@ -22,6 +22,9 @@
 import ch.epfl.sweng.swenggolf.profile.ProfileActivity;
 import ch.epfl.sweng.swenggolf.profile.User;
 
+/**
+ * Class which Convert activities into Fragments.
```

Comment:
It don't really do that. It just a fragment that is used as a base for all other fragments. I don't know why it has this name. @udamiano should be able to answer this question as he created this class.

Reply:
The description would be more : "Class adding functionalities to work with the MainMenuActivity and automatize initialization such as setting the toolbar. also adds functionalities to create initialized fragments such as ShowOfferActivity or CreateOfferActivity."

Reply:
Thanks